source/install/cloud-changelog.md

```diff
@@ -4,12 +4,44 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
 
 Latest Mattermost Cloud releases:
 
+- [Release 2022-12-08](#release-2022-12-08)
 - [Release 2022-12-01](#release-2022-12-01)
 - [Release 2022-11-24](#release-2022-11-24)
 - [Release 2022-11-17](#release-2022-11-17)
 - [Release 2022-11-10](#release-2022-11-10)
 - [Release 2022-10-27](#release-2022-10-27)
-- [Release 2022-10-20](#release-2022-10-20)
+
+## Release 2022-12-08
+
+### Highlights
+
+#### Message Priority and Acknowledgments (Professional plan)
+ - Added message priority labels to the Threads view.
+ - Added support for enterprise users to request acknowledgements on posts and to acknowledge posts.
+
+#### Global Drafts
+ - Added a centralized Drafts view for draft messages.
+
+### Improvements
+
+#### User Interface (UI)
+ - Updated prepackaged version of plugins affected by React 17 upgrade.
+
+#### Administration
+ - The export file now contains the server version and a creation timestamp.
+ - On the purchase modal, admins are now able to buy a subscription for a user count that exceeds the current user count.
```

Comment:
This is also for Cloud Annual Subscription - we should include this as a highlighted new feature in the Cloud 15th release

Reply:
Shouldn't we denote that the ability to buy a "subscription for a user count that exceeds the current user count" only for cloud annual and self-hosted annual? Cloud monthly does not have this capability

Reply:
I see, I'll remove this from this Cloud release for now and will add back for Dec 15th changelog.